fragmentmerge-maven-plugin/src/main/java/org/jaffa/plugins/FragmentMergeMojo.java

```diff
@@ -122,38 +119,130 @@ public void execute() throws MojoExecutionException {
                 targetDirectory = new File(project.getBuild().getDirectory());
             }
 
-            File resources = new File(classesDirectory + META_INF_LOCATION + PROPERTIES_FILE);
+            File applicationResources = new File(classesDirectory + META_INF_LOCATION + PROPERTIES_FILE);
             File dwr = new File(classesDirectory + META_INF_LOCATION + DWR_FILE);
             File jawr = new File(classesDirectory + META_INF_LOCATION + JAWR_FILE);
```

Comment:
I have noticed one thing. The jawr is in html source folder which hasn't picked up by the plugin. Can you check?